packages/server/src/middlewares/authenticate.ts

```diff
@@ -15,17 +15,18 @@ export const extractSessionId = (ctx: Context): string | null => {
 
 export const authenticate = async (ctx: Context, next: () => void) => {
   const sessionId = extractSessionId(ctx);
+
   if (!sessionId) {
-    ctx.body = { error: "Unauthorized", error_description: "no_session_id" };
     ctx.status = 401;
+    ctx.message = "Unauthorized - No sessionId";
```

Comment:
you should reuse the getAccount method here, and give the authenticated user to the context. This way, the other middleware functions have directly access to the authenticated user.
Also, we should define a new router `AuthenticatedRouter`, where we could put all the protected routes